extensions/azurePublishNew/src/recoilModel/atoms/resourceConfigurationState.ts

```diff
@@ -40,12 +40,17 @@ export const userInfoState = atom<UserInfo>({
   default: undefined,
 });
 
+export const operatingSystemState = atom<string>({
```

Comment:
nit:  appServiceOperatingSystemState would avoid confusion.  Others might assume this is tracking the user's OS value.